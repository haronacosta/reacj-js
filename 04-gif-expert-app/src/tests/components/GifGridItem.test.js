import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Pruebas en el componente GifGridItem', () => {
  const title = 'Titulo de prueba';

  const url = 'https://media.giphy.com/media/Js7k1D3Jenq3MUp65Z/giphy.gif';

  let wrapper = shallow(<GifGridItem title={title} url={url} />);

  test('Prueba del componente, traer el componente correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
