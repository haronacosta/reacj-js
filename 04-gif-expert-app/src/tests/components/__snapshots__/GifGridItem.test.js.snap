// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente GifGridItem Prueba del componente, traer el componente correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn animate__delay-1s"
>
  <img
    alt="Titulo de prueba"
    src="https://media.giphy.com/media/Js7k1D3Jenq3MUp65Z/giphy.gif"
  />
  <p>
    Titulo de prueba
  </p>
</div>
`;
